### ISSUE TITLE
Error when generating spec file

### ISSUE THREAD

**Issue by DuncanFaulkner** [allenhwkim/ngentest#14]
Hi

When I run "ngentest count-down/count-down.component.ts > count-down/count-down.component.spec.ts" or "ngentest count-down/count-down.component.ts -s"

I get the following error (see below)

```
Error:
(node:5508) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '1' of null
    at forEach.method (/home/integra/.nvm/versions/node/v10.15.0/lib/node_modules/ngentest/src/lib/parse-typescript.js:71:85)
    at Array.forEach (<anonymous>)
    at parseTypescript (/home/integra/.nvm/versions/node/v10.15.0/lib/node_modules/ngentest/src/lib/parse-typescript.js:65:25)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
(node:5508) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:5508) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

The code file looks like this

imports .....
```
@Component({
  selector: 'edge-count-down',
  templateUrl: './count-down.component.html',
  styleUrls: ['./count-down.component.scss'],
  encapsulation: ViewEncapsulation.None
})
export class CountDownComponent implements OnInit, OnChanges {
  @Input() seconds: number;
  @Input() name: string;
  countdown: string;

  constructor() {}

  ngOnInit() {}

  ngOnChanges(changes: SimpleChanges) { }

  startCountdownTimer() {
    detail removed for brevity...  
  }
}
```

**Comment by DuncanFaulkner** [comment]
Update: it seems to have an issue generating spec files where the file name has a "-" in the name (though this doesn't appear to be an issue when generating spec files for services).

**Comment by allenhwkim** [comment]
Thanks for reporting this. It's fixed with version 0.4.1

**Comment by DuncanFaulkner** [comment]
cheers
